### PR TITLE
feat(packages/sui-react-web-vitals): add visibilityState property

### PR DESCRIPTION
**Pull request by nucliweb**

### CODE REVIEW

**Review by jordevo — APPROVED** [review]

**Review by andresz1 — APPROVED** [review]

File: packages/sui-react-web-vitals/src/index.js [hunk 1]
```diff
@@ -113,7 +113,8 @@ export default function WebVitalsReporter({
         path: hasPathOnRoute ? pathname : getNormalizedPathname(pathname),
         target,
         loadState: attribution.loadState,
-        ...(attribution.eventType && {eventType: attribution.eventType})
+        ...(attribution.eventType && {eventType: attribution.eventType}),
+        visibilityState: document.visibilityState
```
Comment (jordevo) [inline packages/sui-react-web-vitals/src/index.js:117]: I'm assuming `ducment` will be available because we're inside a `useMount`

**Review by oriolpuig — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

File: packages/sui-react-web-vitals/src/index.js (hunk 1 above)
Comment (nucliweb) [reply packages/sui-react-web-vitals/src/index.js:117]: Yes, we have access to document inside a `useMount`

**Comment by jordevo** [comment]
I reckon it'd be great to add some documentation on the package's readme so that's easy to know the new stuff that's available.

**Comment by nucliweb** [comment]
> I reckon it'd be great to add some documentation on the package's readme so that's easy to know the new stuff that's available.

I agree; first, we want to test if this info will be useful.

We are looking for quick wins to improve the INP metric.

**Pull request merged**